tests/testthat/test-filters.R

```diff
@@ -191,8 +191,7 @@ test_that("error thrown when tree can't be constructed", {
 })
 
 test_that("naomi IDs can be mapped to hint IDs", {
-  expect_equal(get_hint_id(2), "prevalence")
-  expect_equal(get_hint_id("2"), "prevalence")
+  expect_equal(get_hint_id("prevalence"), "prevalence")
```

Comment:
@r-ash -- this change is because the indicators metadata table is updated with `indicator` rather than `indicator_id` ([here](https://github.com/mrc-ide/naomi/commit/69e880561832a8fbf42d4c9ab65dd7e00b637b99)). Can you review this change and make sure it is the right thing to do and flag if any implications for hint?

Reply:
This function and `get_indicator_display_name` aren't actually called from anywhere - must be old helpers which never got removed. I've removed them now